tests/e2e/test_rosacli_oidc_config.go

```diff
@@ -228,6 +238,13 @@ var _ = Describe("Register ummanaged oidc config testing",
 				"--prefix", accountRolePrefix,
 				"-y")
 			Expect(err).To(BeNil())
+
+			By("Delete temp dir")
+			if dirToClean != "" {
+				_, err := rosaClient.Runner.RunCMD([]string{"rm", "-rf", dirToClean})
+				Expect(err).To(BeNil())
+			}
```

Comment:
Why not use `err := os.RemoveAll(tempfile)`?

Reply:
After discussion, remove this step to leave the temp dir cleanup by job itself